opencontext/server/routes/settings.py

```
@@ -95,22 +107,39 @@ async def get_model_settings(
     except HTTPException:
         raise
     except Exception as e:
-        logger.exception(f"获取模型设置失败: {e}")
+        logger.exception(f"Failed to get model settings: {e}")
```

Ruff (0.13.3)

Redundant exception object included in `logging.exception` call

(TRY401)